lib/node_modules/@stdlib/blas/ext/base/dnansumkbn/lib/index.js

```diff
@@ -38,7 +38,7 @@
 *
 * var x = new Float64Array( [ 2.0, 1.0, 2.0, -2.0, -2.0, 2.0, 3.0, 4.0, NaN, NaN ] );
 *
-* var v = dnansumkbn.ndarray( 5, x, 2, 1 );
+* var v = dnansumkbn.ndarray( x.length, x, 2, 1 );
```

Comment:
```suggestion
* var v = dnansumkbn.ndarray( 5, x, 2, 1 );
```

This needs to remain `5`, as the stride equals `2`.